src/spark/wallet.cpp

```diff
@@ -0,0 +1,1355 @@
+#include "wallet.h"
```

Comment:
We already have not one but two files names `wallet.cpp` (and `wallet.h` too). Can we rename it to `sparkwallet`?

Reply:
agree, fixed